index.html

```diff
@@ -13,196 +13,217 @@
-                <li class="social-list"><a href="#"><img src="images/medium.png" alt="medium"></a></li>
-                <li class="social-list"><a href="#"><img src="images/github.png" alt="github"></a></li>
-                <li class="social-list"><a href="#"><img src="images/angellist.png" alt="angellist"></a></li>
-            </ul>
-            <div class="work-button"><a class="project-button" href="#">Get My Resume</a></div>
-            <div class="language-box1">
-                <h3 class="canopy">Languages</h3>
-                <a href="#"><img class="icons" src="images/down-arrow.png" alt="arrow"></a>
-            </div>
-            <ul>
-                <li class="language">
-                    <img src="images/Ellipse1.png" alt="js">
-                    <h3 class="language-text">JavaScript</h3>
-                </li>
-                <li class="language">
-                    <img src="images/Ellipse2.png" alt="js">
-                    <h3 class="language-text">HTML</h3>
-                </li>
-                <li class="language">
-                    <img src="images/Ellipse3.png" alt="js">
-                    <h3 class="language-text">CSS</h3>
-                </li>
-            </ul>
-            <div class="language-box">
-                <h3 class="canopy">Frameworks</h3>
-                <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
+            <div class="about">
+                <h1 class="card-title">About <br>Myself</h1>
+                <p>I’m a software developer! I can help you build a product, feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.</p>
+                <h2>LET’S CONNECT</h2>
+                <ul>
+                    <li class="social-list"><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+                    <li class="social-list"><a href="#"><img src="images/Linkedin.png" alt="twitter"></a></li>
+                    <li class="social-list"><a href="#"><img src="images/medium.png" alt="medium"></a></li>
+                    <li class="social-list"><a href="#"><img src="images/github.png" alt="github"></a></li>
+                    <li class="social-list"><a href="#"><img src="images/angellist.png" alt="angellist"></a></li>
+                </ul>
+                <div class="work-button"><a class="project-button" href="#">Get My Resume</a></div>
             </div>
-            <div class="language-box">
-                <h3 class="canopy">Skills</h3>
-                <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
+            <div class="languages-box2">
+                <div class="language-box1">
+                    <h3 class="canopy">Languages</h3>
+                    <a href="#"><img class="icons" src="images/down-arrow.png" alt="arrow"></a>
```

Comment:
- [ ] Kindly ensure that the arrows for the Languages, Frameworks and Skills are dropdown menus that display in detail the examples of Languages, Frameworks and Skills based on the section.
- [ ] Within each drop-down menu, ensure that there are up to 5 tags per row listing the technologies.

Reply:
I kindly request that you have a look the the template 1 that I was doing, they're a no rows of technologies that have 5 tags.